Sources/Clappr/Classes/Plugin/Playback/AVFoundationPlayback.swift

```diff
@@ -507,12 +524,16 @@ open class AVFoundationPlayback: Playback {
     }
 
     func handleDvrAvailabilityChange() {
-        if lastDvrAvailability != isDvrAvailable {
+        if dvrAvailabilityChanged {
             trigger(.didChangeDvrAvailability, userInfo: ["available": isDvrAvailable])
             lastDvrAvailability = isDvrAvailable
         }
     }
 
+    private var dvrAvailabilityChanged: Bool {
+        return lastDvrAvailability != isDvrAvailable && playbackType == .live
```

Comment:
I think that we can change the order because the playbackType is more important than DVR availability.

Reply:
Done.